deploy-help/deploy.sh

```diff
@@ -0,0 +1,41 @@
+#! /bin/bash -e
+
+deployment_dir=~/deployment
```

Comment:
Please make it a parameter. Make it optional and set it to `~/deployment/` by default.